src/qmapshack/gis/trk/CToRouteNameProjectDialog.h

```diff
@@ -0,0 +1,86 @@
+/**********************************************************************************************
+    Copyright (C) 2021 Oliver Eichler <oliver.eichler@gmx.de>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+**********************************************************************************************/
+
+#ifndef CTOROUTENAMEPROJECTDIALOG_H
+#define CTOROUTENAMEPROJECTDIALOG_H
+
+#include "gis/prj/IGisProject.h"
+#include "gis/CGisWorkspace.h"
+#include "ui_IToRouteNameProjectDialog.h"
+#include <QDialog>
+
+class QTreeWidget;
+
+/*
+ * Dialog Class for the track to route conversion.
+ */
+class CToRouteNameProjectDialog : public QDialog, private Ui::IToRouteNameProjectDialog
```

Comment:
Wouldn't something like `CTrkToRteDialog`  be more expressive on the purpose of that dialog?